release/_publishing.sh

```diff
@@ -36,19 +36,47 @@ function init_gcs {
 }
 
 function update_latest_in_bundles {
-    local file_path="${BASE_DIR}/bundles.yml"
+	local product_version_key=${_PRODUCT_VERSION%-*}
```

Comment:
@kiwm, can you please try to use a code similar to this [one](https://github.com/brianchandotcom/liferay-docker/pull/628/commits/0a08213c374a94b7770d8fe9954888524cebb07d) instead?

`${_PRODUCT_VERSION%-*}` isn't so intuitive to understand and it may not handle appropriately all cases, like `2024.q1.1, 7.4.13-u120, 7.4.3.120-ga120, etc`